<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>megapixel</class>
 <widget class="QWidget" name="megapixel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>megapixel</string>
  </property>
  <widget class="QPushButton" name="pushButtonOpenSource">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Source</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStart">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Encoding</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>771</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Queue</string>
    </attribute>
    <widget class="QListWidget" name="listWidgetQueue">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>761</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Encoder:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxEncoders">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>131</width>
       <height>36</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>avif</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>webp</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelAvifMinQ">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Min Q:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxAvifMinQ">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>61</width>
       <height>36</height>
      </rect>
     </property>
     <property name="maximum">
      <number>63</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelAvifMaxQ">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Q:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxAvifMaxQ">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>61</width>
       <height>36</height>
      </rect>
     </property>
     <property name="maximum">
      <number>63</number>
     </property>
     <property name="value">
      <number>40</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelAvifThreads">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Threads:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxAvifThreads">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>61</width>
       <height>36</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelAvifSpeed">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxAvifSpeed">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>220</y>
       <width>61</width>
       <height>36</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelAvifDepth">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Depth:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxAvifDepth">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>61</width>
       <height>36</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelAvifColorFormat">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Color Format:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxAvifColorFormat">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>61</width>
       <height>36</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>420</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>422</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>444</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelAvifRange">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>120</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Range:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxAvifRange">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>120</y>
       <width>81</width>
       <height>36</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>full</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>limited</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelAvifTileRows">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tile-Rows:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxAvifTileRows">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>61</width>
       <height>36</height>
      </rect>
     </property>
     <property name="maximum">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelAvifTileColumns">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tile-Columns:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxAvifTileCols">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>61</width>
       <height>36</height>
      </rect>
     </property>
     <property name="maximum">
      <number>6</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveTo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save to...</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOutput">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxBatchAdd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Batch Add Pictures</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
